### PR TITLE
Add kubnetes features page

### LINKED ISSUE #3361: Beta: Build Kubernetes features page
Copy doc: https://docs.google.com/document/d/1mEOBW4pfsWJqTLm9Pn9myKtaslD-QB2RvUKcNQ52bXU/edit

### PR DESCRIPTION
**Pull request by steverydz**
## Done

Add `/kubernetes/features`

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: <http://0.0.0.0:8001/kubernetes/features>
- See that is matches the [copy doc](https://docs.google.com/document/d/1mEOBW4pfsWJqTLm9Pn9myKtaslD-QB2RvUKcNQ52bXU/edit#)

## Issue / Card
Fixes #3361

### CODE REVIEW

File: templates/kubernetes/features.html [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+{% block title %}Features of Canonical Kubernetes on Ubuntu{% endblock %}
+{% block meta_description %}Canonical OpenStack on Ubuntu includes software-defined networking and storage options, architectural flexibility, and shared community-driven ops code independent of architecture.{% endblock meta_description %}
+
+{% block content %}
+
+<section class="p-strip is-deep is-bordered">
+  <div class="row">
+    <div class="col-8">
+      <h1>Canonical Distribution of Kubernetes features</h1>
+      <p>Architectural freedom. Fully automated operations. Extensible Kubernetes for all.</p>
```
Comment (anthonydillon) [inline templates/kubernetes/features.html]: I think this one should be a `p-heading--four`

File: templates/kubernetes/features.html [hunk 2]
```diff
@@ -0,0 +1,160 @@
...
+  </div>
+</section>
+
+<section class="p-strip is-bordered">
+  <div class="row">
+    <div class="col-8">
+      <h2>Networking</h2>
+      <p>Kubernetes imposes fundamental requirements on your networking implementation including:</p>
+      <ul class="p-list">
+        <li class="p-list__item is-ticked">all containers can communicate with all other containers without NAT</li>
```
Comment (anthonydillon) [inline templates/kubernetes/features.html]: Can you uppercase the first letter please.

File: templates/kubernetes/features.html [hunk 3]
```diff
@@ -0,0 +1,160 @@
...
+        <li class="p-list__item is-ticked">all nodes can communicate with all containers (and vice-versa) without NAT</li>
+        <li class="p-list__item is-ticked">the IP that a container sees itself as is the same IP that others see it as</li>
+      </ul>
+    </div>
+  </div>
+  <div class="row">
+    <div class="col-8">
+      <p>Kubernetes uses Container Network Interface (CNI) as an interface between network providers and Kubernetes networking. CNI is a library definition and a set of tools, under the umbrella of the Cloud Native Computing Foundation project.</p>
+      <p>Canonical&rsquo;s Distribution of Kubernetes comes pre-packaged with several tested CNI plugins like Calico and Flannel. Our distribution of Kubernetes is open and extensible &mdash; bring your favorite CNI plugin and extend CDK. If you require assistance, please take advantage of one of our consulting packages.</p>
```
Comment (anthonydillon) [inline templates/kubernetes/features.html]: Looks like the sentence "If you require assistance, please take advantage of one of our consulting packages." has been removed from the copy doc.

File: templates/kubernetes/features.html (hunk 1 above)
Comment (steverydz) [reply templates/kubernetes/features.html]: Done

File: templates/kubernetes/features.html (hunk 2 above)
Comment (steverydz) [reply templates/kubernetes/features.html]: Done

File: templates/kubernetes/features.html (hunk 3 above)
Comment (steverydz) [reply templates/kubernetes/features.html]: Done

**Comment by steverydz** [comment]
@anthonydillon Requested changes made

File: navigation.yaml [hunk 4]
```diff
@@ -75,6 +75,8 @@ containers:
       path: /kubernetes/install
     - title: Kubernetes partners
       path: /kubernetes/partners
+    - title: Kubernetes features
```
Comment (pmahnke) [inline navigation.yaml]: can you make these all just the final word, no Kubernetes?

Install, Managed, Partners, Features, etc?

Sorry

Comment (steverydz) [reply navigation.yaml]: Done

**Comment by frankiegranato** [comment]
- [ ] We're adding a 'CNI plugins' heading to help break up the large chunk.
- [ ] Replace image at the top of the page

**Comment by steverydz** [comment]
@frankiegranato Changes done

**Comment by anasereijo** [comment]
@steverydz 

screencapture-ubuntu-pr-3454-run-demo-haus-kubernetes-features-2018-06-26-17_05_06

**Comment by frankiegranato** [comment]
Can we amend the ordering in the secondary nav like so :
kubernetes menu ordering

**Comment by frankiegranato** [comment]
This is not done yet: 
- [ ] We're adding a 'CNI plugins' heading to help break up the large chunk.

**Comment by steverydz** [comment]
@anasereijo @frankiegranato Design and UX updates done

**Comment by frankiegranato** [comment]
- [ ] Image text should be "caption style" (h6). See comment in doc.

**Pull request merged**